components/Toolbar/BlueprintsToolbar.js

```diff
@@ -34,6 +34,26 @@ const BlueprintsToolbar = props => (
         >
           <FormattedMessage defaultMessage="Create Blueprint" />
         </button>
+        <div className="dropdown btn-group dropdown-kebab-pf">
+          <button
+            className="btn btn-link dropdown-toggle"
+            type="button"
+            id="dropdownKebab"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false"
+          >
+            <span className="fa fa-ellipsis-v" />
+          </button>
+          <ul className="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebab">
+            <li><a href="#" onClick={(e) => props.handleShowModalManageSources(e)}>
+              <FormattedMessage
+                defaultMessage="View Source"
```

Comment:
Singular here, even though it's plural elsewhere. Should be View Sources to match with the dialog.